styles.css

```diff
@@ -0,0 +1,240 @@
+.container {
+  width: 375px;
+  display: flex;
+  justify-content: center;
+  padding: 25px;
+  margin-left: 25px;
+}
```

Comment:
- [ ] The design of the page is like in center, not fitting the view 
![bgbg](https://user-images.githubusercontent.com/61562626/129426421-183958b7-a079-4b20-9764-bf9174577d16.jpeg)